recipes/invoke/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.12.2" %}
+
+package:
+  name: invoke
+  version: {{ version }}
+
+source:  
+  fn: invoke-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/i/invoke/invoke-{{ version }}.tar.gz
+  md5: f46192ea664325464a02f1c7f735e3cb
+
+build:
+  number: 0
+  script: python setup.py install
```

Comment:
Right, I thought this script element was just a shortcut for not needing the build.bat and build.sh files. So you want me to change the script line into: `python setup.py install --single-version-externally-managed --record=record.txt` then?